json-rpc/Cargo.toml

```diff
@@ -23,6 +23,7 @@ tokio = { version = "1.1.0", features = ["full"] }
 warp = { version = "0.3.0", features = ["tls"] }
 reqwest = { version = "0.11.0", features = ["blocking", "json"], default_features = false, optional = true }
 proptest = { version = "0.10.1", optional = true }
+serde_derive = "1.0.117"
```

Comment:
I'd probably just enable the `derive` feature on the `serde` crate instead of adding this directly.